provisioning/roles/nginx/templates/letsencrypt.j2

```diff
@@ -0,0 +1,15 @@
+# Initial configuration for Let's Encrypt.
+# You should not use this configuration as the definitive one.
```

Comment:
Could we have instead two templates, one for the HTTP website and one for the HTTPS website. The HTTP one would be enabled first, then letsencrypt.sh would be used to generate certificates, then the HTTPS one would be enabled?